### PR TITLE
[CDAP-18069] Test connection during connection creation with new API

### PR DESCRIPTION
**Pull request by njbriggs**
Jira: https://cdap.atlassian.net/browse/CDAP-18069

Test in progress:
image

Test success:
image

Test failed:
image

Server-side error during test:
image

### CODE REVIEW

File: app/cdap/components/Connections/Create/ConnectionConfiguration/ConnectionConfigForm.tsx [hunk 1]
```diff
@@ -27,17 +29,30 @@ const useStyle = makeStyle(() => {
...
+    connectionTestMessage: {
+      display: 'flex',
+      height: '2rem',
+      padding: '0 10px',
+    },
+    connectionTestSuccess: {
+      color: '#4ab63c',
+    },
+    connectionTestFailure: {
+      color: '#d15668',
```
Comment (elfenheart) [inline app/cdap/components/Connections/Create/ConnectionConfiguration/ConnectionConfigForm.tsx]: are these not theme colors?

**Review by elfenheart — COMMENTED** [review]
also curious, is there no field level errors like in pipelines?

File: app/cdap/components/Connections/Create/ConnectionConfiguration/ConnectionConfigForm.tsx (hunk 1 above)
Comment (njbriggs) [reply app/cdap/components/Connections/Create/ConnectionConfiguration/ConnectionConfigForm.tsx]: Looks like the failure color is `red-03`. The green color is not in the theme but it looks like they're not in sorted order so I'll add it.

**Comment by njbriggs** [comment]
Using colors from theme now.

**Review by elfenheart — APPROVED** [review]
Just question around whether there is field level error or not? From the screenshots it seems like it's only a global error

**Pull request merged**